doc/guides/collaborator-guide.md

```diff
@@ -791,4 +820,5 @@ If you cannot find who to cc for a file, `git shortlog -n -s <file>` can help.
 [git-username]: https://help.github.com/articles/setting-your-username-in-git/
 [node-core-utils-credentials]: https://github.com/nodejs/node-core-utils#setting-up-credentials
 [node-core-utils-issues]: https://github.com/nodejs/node-core-utils/issues
+[security reporting]: https://github.com/nodejs/SECURITY.md
```

Comment:
This is a 404 and should be 
```suggestion
[security reporting]: https://github.com/nodejs/node/blob/master/SECURITY.md
```
or
```suggestion
[security reporting]: ../../SECURITY.md
```

Reply:
(please always use `HEAD` instead of `master`)